CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # New Relic Ruby Agent Release Notes
 
+## Upcoming Release
+
+This upcoming release of the agent adds instrumentation for Active Support caching operations.
+
+- **Add Active Support Instrumentation**
+
+  Active Support instruments caching-related operations. This change includes the instrumentation in the agent. A new segment is created when one of the operations is called. The key, store, and other attributes will be included on the segment as params. [PR#1742](https://github.com/newrelic/newrelic-ruby-agent/pull/1742)
+
+  | Configuration name | Default | Behavior |
+  | ----- | ----- | ----- |
+  | `disable_active_support` | `false` | If `true`, disables ActiveSupport instrumentation. |
```

Comment:
"Active Support" and "ActiveSupport" are both correct depending on context, but seeing them side by side in the same file like this makes me wonder if we should standardize on one or the other.

Reply:
Oof, great point. I usually see "Active Support" in the Rails guides, so I was trying to follow that. I missed the config spelling though. Are you comfortable with the spaced version?

Reply:
The spaced version has been incorporated in this commit: 278635542852a978fcb57364752500c445f237c6

Happy to revert.

Reply:
Oof. This reference was fixed in this commit: 4c10f14f713ef094a86497628a9b864a2e1d1a92